### DISCUSSION TITLE
unknown argument ignored in clang-cl: '-cbstring' [-Werror,-Wunknown-argument]	MyDriver1

### DISCUSSION THREAD

**Discussion by John-Ming-yang** [NewWorldComingSoon/llvm-msvc-issues#53] [Q&A]
unknown argument ignored in clang-cl: '-cbstring' [-Werror,-Wunknown-argument]	MyDriver1
vs2022
生成时提示我这个错误

**Comment by gmh5225** [comment]
Enable ``/WX-`` or upload your sample project
